src/test/scala/io/iohk/ethereum/ObjectGenerators.scala

```diff
@@ -1,7 +1,7 @@
 package io.iohk.ethereum
 
 import java.math.BigInteger
-import org.scalacheck.{Arbitrary, Gen}
+import org.scalacheck.{Arbitrary, Gen, _}
```

Comment:
You are right. As we are only using `org.scalacheck.{Arbitrary, Gen}` I'll change it to that.